release-content/0.14/release-notes/12013_Upstreaming_bevy_color.md

```diff
@@ -1 +1,31 @@
-TODO
+Colors are a huge part of building a good game: UI, effects, shaders and more all need fully-feature, correct and convenient color tools.
+Bevy now supports a wide range of color spaces, each with their own type (e.g. `LinearRgba`, `Hsla` or `Oklaba`),
+and offers a wide range of fully documented operations on and conversions between them.
+
+Critically, color mixing / blending is now supported: perfect for procedurally generating color palettes and working with animations.
+
+```rust
+use bevy_color::{palettes::tailwind, Mix};
```

Comment:
I noticed that in this example we are importing a pallete and a trait, but not the colorspaces (like Srgba, Oklcha... ), can this be a problem?

Reply:
That *is* correct, maybe we should just use a glob import here.

Reply:
```suggestion
use bevy_color::*;
```